src/components/posts/GenderGraphContainer.js

```diff
@@ -0,0 +1,82 @@
+import React from "react";
+import { LineChart, Line, XAxis, YAxis, CartesianGrid, Tooltip, ResponsiveContainer } from 'recharts';
+
+export default function GenderGraphContainer() {
+
+    const data = [
+        {
+          name: '1965',
```

Comment:
you can probably abstract this hardcoding of data into a function with an array representing percentage values and simply incrementing the name component by 5 each time to clean-up how much hardcoding you have to do.